src/widgets/Seperator.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/no-unused-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import React from 'react';
+import { View } from 'react-native';
+import PropTypes from 'prop-types';
+
+import { GREY } from '../globalStyles/index';
+
+/**
+ * Simple Separator display component rendering a line.
+ *
+ * @prop {Number} width            The width of the seperator
+ * @prop {String} length           The length of the seperator as a percentage string.
+ * @prop {Number} marginBottom     Number of pixels for the bottom margin.
+ * @prop {Number} marginTop        Number of pixels for the bottom margin.
+ * @prop {Number} marginHorizontal Number of pixels for the top & bottom margin.
+ */
+export const Seperator = props => {
+  const { mainContainerStyle, borderContainerStyle } = localStyles(props);
+  return (
+    <View style={mainContainerStyle}>
```

Comment:
I could be wrong, but you may be able to do this component without a container view? Miss naming the style might support this notion 😄

Reply:
That said, given it's just a border, couldn't you just put a border under the label container?

Reply:
- Need the container to be able to do non full length like:
![image](https://user-images.githubusercontent.com/35858975/71218383-2e419200-2326-11ea-8bdc-4a74756dc3fa.png)

- In my opinion we really need more smaller components to reuse. I'm quite frequently writing the same things where having generic display components would help a lot. It's incredibly noticeable the benefits when working in a project which has a decent library of simple reusable presentation components v this project where there are minimal!

- For this component specifically, it's seems to be a lot easier to chuck in a little `<Seperator/> when you need a quick grouping line rather than messing around with styles.. padding/margins etc etc and, making it work for all uses of the component

Reply:
I see, generally agree with having a library of reusable small components in the repo, and fair enough